wfc_gh/WaveFunctionCollapse.cs

```diff
@@ -659,40 +618,50 @@ internal class Native
                                                                      IntPtr source_wfc_world_state_handle);
 
         [DllImport("wfc", CallingConvention = CallingConvention.StdCall)]
-        internal static unsafe extern void wfc_world_state_clone_from(IntPtr destination_wfc_world_state_handle,
-                                                                      IntPtr source_wfc_world_state_handle);
+        internal static unsafe extern WfcWorldStateCloneFromResult wfc_world_state_clone_from(IntPtr destination_wfc_world_state_handle,
+                                                                                              IntPtr source_wfc_world_state_handle);
 
         [DllImport("wfc", CallingConvention = CallingConvention.StdCall)]
         internal static extern void wfc_world_state_free(IntPtr wfc_world_state_handle);
 
         [DllImport("wfc", CallingConvention = CallingConvention.StdCall)]
-        internal static unsafe extern WfcWorldStateSlotsSetResult wfc_world_state_slots_set(IntPtr wfc_world_state_handle,
-                                                                                            SlotState* slots_ptr,
-                                                                                            UIntPtr slots_len);
-
+        internal static extern WfcWorldStateSlotModuleSetResult wfc_world_state_slot_module_set(IntPtr wfc_world_state_handle,
+                                                                                                ushort pos_x,
+                                                                                                ushort pos_y,
+                                                                                                ushort pos_z,
+                                                                                                ushort module,
+                                                                                                uint value);
```

Comment:
what exactly is `value`? is is the bool that marks whether the module is allowed?

Reply:
Yes! Booleans do not exist in C APIs, so people usually use uints or bitflags.

Reply:
I wanted to point out that it is not clear from the word `value`.